server/chat-plugins/chatlog.ts

```diff
@@ -492,13 +511,19 @@ const LogSearcher = new class {
 				'-e', `[^a-zA-Z0-9]${search.split('').join('[^a-zA-Z0-9]*')}([^a-zA-Z0-9]|\\z)`,
 				`${__dirname}/../../logs/chat/${roomid}`,
 				'-C', '3',
+				'-m', `${MAX_RESULTS}`,
 			];
 			output = await execFile('rg', options, {maxBuffer: Infinity, cwd: path.normalize(`${__dirname}/../`)});
 		} catch (error) {
 			if (error.message.includes('Command failed')) return LogViewer.error(`No results found.`);
 			return LogViewer.error(`${error.message}`);
 		}
-		return this.render(output.stdout.split('--'), roomid, search, limit);
+		return this.render(
+			output.stdout.split('--').reverse().slice(0, MAX_RESULTS),
```

Comment:
Do you need to slice to `MAX_RESULTS` if your already limiting it in your ripgrep query?

Reply:
Nope. forgot to remove.